### ISSUE TITLE
Overflow menu stays open but blank after tapping push notification

### ISSUE THREAD

**Issue by rachelmcr** [woocommerce/woocommerce-android#1949]
**Describe the bug**

If I had the overflow menu (Settings/Help) open the last time I used the app, and then I tap a push notification to open the app again, the push notification takes me to the correct place in the app (a new order or review)— but a blank overflow menu appears on screen.

**To Reproduce**
1. Tap the overflow menu to open it (to see the Settings/Help & support options)
2. Background the app or lock your device screen.
3. Create a new order or review on your store.
4. Tap the push notification for the new order/review when it arrives.

Result: The overflow menu stays open but appears blank.

**Screenshots**

Order details with overflow|Review details with overflow
-|-
![Screenshot_20200214-165650](https://user-images.githubusercontent.com/8658164/74551983-2c720480-4f4c-11ea-9e0a-e93c32c4082f.png)|![Screenshot_20200214-170131](https://user-images.githubusercontent.com/8658164/74551975-2a0faa80-4f4c-11ea-98eb-a2055308e430.png)

**Expected behavior**

I would expect the overflow menu to close when I'm taken to the new screen from the push notification.

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: moto e5 play
- Android version: 8.1.0
- WooCommerce Android version: 3.6-rc-1

**Comment by anitaa1990** [comment]
We are no longer displaying the overflow menu in detail screens, (only in the top screens). I tried to reproduce this in the latest version of the app and I am no longer able to. @rachelmcr can you confirm if this is still an issue please? Thank you 🙇‍♀️

**Comment by rachelmcr** [comment]
I can still reproduce the issue using the original steps. I tested again on moto e5 play, Android 8.1.0, app version 4.3-rc-1.

Even though the overflow menu isn't available in detail screens, if you open the overflow menu in a screen where it's available, the blank menu is visible in details screens after tapping on a push notification. Here are the exact steps I used to reproduce this time:

1. Opened the review list screen.
2. Opened the overflow menu.
3. Put the app in the background.
4. Created a new review on the store.
5. Tapped the push notification, which opened the review detail screen with a blank white box in the top right.
